### DISCUSSION TITLE
Authentication problem in iOS

### DISCUSSION THREAD

**Discussion by opanyki314** [Hieromon/AutoConnect#517] [Q&A]
Hi! I have a problem with authentication in iOS. When I set config.autoscope = AC_AUTHSCOPE_AC and try to connect to my AP in SoftAP mode, I see an authentication error on my iPhone. So I can't connect to my AP. But on my Android phone it works fine. If I don't set the AC_AUTHSCOPE_AC bit, everything will work fine everywhere. Is there any way I can fix this?

**Comment by Hieromon** [comment]
@opanyki314 
1. What is your iOS version? I will try to reproduce the problem on my iPhone.
2. When does an authentication error occur? Does the authentication error occur when the ESP module opens a captive portal without a WiFi connection to any APs you have? In other words, is it when the web page pops up with the ESP module's AP as esp32ap selected in the iOS WiFi settings screen?
3. Is the above phenomenon the same situation I am referring to in the [documentation](https://hieromon.github.io/AutoConnect/adauthentication.html#authentication-within-the-captive-portal-state)?

**Comment by opanyki314** [comment]
1. iOS version 15.1
2. The error occur when the ESP module opens a captive portal without a WiFi connection to any APs. I select my AP from the list and try to connect. But when I push the connect button the error occurs.
3. It looks very similar but I don't need authentication in the captive portal state.

**Reply by Hieromon** [reply]
@opanyki314 
I have reproduced the issue, it is a bug in AutoConnect. Move this topic to Issue to begin diagnosis and fix.
It doesn't seem to be a difficult fix, so I will release a patch soon as v1.3.7 with the fix.
Thank you for reporting back.

**Reply by opanyki314** [reply]
OK. Thank you for your great job.
